python/setup_project/storage_locations_page.py

```diff
@@ -95,7 +95,9 @@ def os_key(element):
 
             # don't allow editing data from Shotgun, too dangerous
             if store_info["defined_in_shotgun"] and store_info[os_key]:
-                tooltip = "Can not edit paths defined Shotgun. This has to be changed in Shotgun's preferences."
+                tooltip = ("There is already a secondary Local Storage defined in your Shotgun Site Preferences "
+                           "named \"secondary\". In order to change these paths, you'll need to update this Local "
+                           "Storage definition in your Shotgun Site Preferences.")
```

Comment:
The first "secondary" should be removed so that it reads "There is already a Local Storage defined..." and the second "secondary" should be ```store_name``` rather than a fixed string.